### PR TITLE
Chrisdias/updates

### PR DESCRIPTION
**Pull request by chrisdias**

### CODE REVIEW

**Comment by bowdenk7** [comment]
You want to deploy it to an app service so we can see a preview? 😃

**Comment by bowdenk7** [comment]
If only that was a feature *cough*

**Comment by chrisdias** [comment]
https://cdias-test-tas-express.azurewebsites.net/

**Comment by bowdenk7** [comment]
looks good full screen, but it doesn't shrink well:
image

Does it look bad stacked vertically?

**Comment by chrisdias** [comment]
ya, any help with resizing would be appreciated :).  I did't try vertically, because I wanted to keep it all above the fold.  what do you think?

**Comment by bowdenk7** [comment]
ya I like having it all in one glance, I'll add some resize css to stack when it's too skinny and we'll see how it looks

**Comment by misolori** [comment]
My initial impression of the app is that it's very overwhelming, which leads to me feel like it's bloated. Is there any way to reduce the amount of content? Both in copy and images. Will leave some more specific feedback in the code.

File: public/style.css [hunk 1]
```diff
@@ -58,9 +61,8 @@ a {
 /* === Content === */
 
 .content {
-  margin: 0 auto;
-  max-width: 1100px;
-  padding: 30px 24px;
+  max-width: fit-content;
```
Comment (misolori) [inline public/style.css]: Why remove the max-width? This makes the content extend to the full width which creates a lot of tension.

File: public/style.css [hunk 2]
```diff
@@ -184,19 +206,46 @@ a {
 }
 
 .numberCircle {
+
```
Comment (misolori) [inline public/style.css]: These circles feel way too large, I'd size them down to be about 32px in size with a 16px font-size for the text.

File: src/template.html [hunk 3]
```diff
@@ -20,124 +20,122 @@
...
 
-    <div class="content-wrapper-light">
-      <div class="content">
-        <div class="column">
-          <div>
-            <p>3. Push with git to redeploy the app using the commands to the right.</p>
+            <div class="numberCircle-Light"></div>
             
+            <div>
+              <p class="title">Get the walkthrough</p>
```
Comment (misolori) [inline src/template.html]: This is out of place with the rest of content. Any reason why this wasn't centered and placed below the content?

File: src/template.html [hunk 4]
```diff
@@ -20,124 +20,122 @@
...
+      <strong id="timeLeft">...</strong>
+      minutes...
     </p>
     <img src="./images/node-logo.svg" alt="Azure App Service">
-    <h1>🎉 Welcome to Express on Azure App Service 🎉</h1>
-    <h2>Hosted at
-      <a href="/" id="host"></a>
-    </h2>
+    <h1> Welcome to Azure! </h1>
+    <p>This is a simple Node application using the popular Express framework, hosted on Azure App Service at <a href="/" id='host'></a>.</p>
```
Comment (misolori) [inline src/template.html]: This feels like unecessary fluff text

File: src/template.html [hunk 5]
```diff
@@ -20,124 +20,122 @@
...
-      </div>
-    </div>
+
 
     <div class="content-wrapper">
       <div class="content">
         <div class="column">
-          <div>
-              <p>2. Make a text change in <b>template.html</b></p>
+          <div class="content">
```
Comment (misolori) [inline src/template.html]: This is the same class being used two levels above, so you're inheriting a lot of duplicate styles.

File: public/style.css (hunk 2 above)
Comment (chrisdias) [reply public/style.css]: fixed

**Comment by chrisdias** [comment]
i've made quite some fixes this morning.  the content doesn't quite align (it's too far right) and i really want a couple of small images in columns 2 and 3 but need to work on that.  https://cdias-test-tas-express.azurewebsites.net/ has been updated.

**Comment by bowdenk7** [comment]
What about this?

screen shot 2018-09-03 at 5 26 06 pm

And it collapses to this, when the window is small:
image

**Pull request merged**